paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/activity/FawryActivity.kt

```diff
@@ -71,18 +72,21 @@ class FawryActivity : AppCompatActivity() {
         private const val EXTRA_BILLING_DETAILS = "external_payment_method_billing_details"
     }
 
-    object FawryConfirmHandler : ExternalPaymentMethodConfirmHandler {
-        override fun createIntent(
-            context: Context,
+    class FawryConfirmHandler(private val context: Context) : ExternalPaymentMethodConfirmHandler {
+
+        override fun confirmExternalPaymentMethod(
             externalPaymentMethodType: String,
-            billingDetails: PaymentMethod.BillingDetails?,
-        ): Intent {
-            return Intent().setClass(
-                context,
-                FawryActivity::class.java
+            billingDetails: PaymentMethod.BillingDetails
+        ) {
+            context.startActivity(
+                Intent().setClass(
+                    context,
+                    FawryActivity::class.java
+                )
+                    .addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
```

Comment:
I'm not sure what the best thing to do here is to make it most consistent with what merchants will do. But typically I don't think we'd launch with new task.

Reply:
Fair, updated this. Now PaymentSheetPlaygroundActivity is the ExternalPaymentMethodConfirmHandler, which I think makes sense for this pattern of launching a new activity.